packages/core/src/env-set/oidc.test.ts

```diff
@@ -48,63 +48,68 @@ describe('oidc env-set', () => {
     readFileSyncSpy.mockRestore();
   });
 
-  it('should read private keys if `OIDC_PRIVATE_KEY_PATH` is provided - [compatibility]', async () => {
-    // Unset the `OIDC_PRIVATE_KEY_PATHS` environment variable config by jest.setup.ts.
+  it('should generate a default OIDC private key if `OIDC_PRIVATE_KEY_PATHS` and `OIDC_PRIVATE_KEYS` are not provided', async () => {
+    process.env.OIDC_PRIVATE_KEYS = '';
     process.env.OIDC_PRIVATE_KEY_PATHS = '';
 
-    process.env.OIDC_PRIVATE_KEY_PATH = 'foo.pem';
+    const readFileSyncSpy = jest.spyOn(fs, 'readFileSync').mockImplementation(() => {
+      throw new Error('Intent read file error');
+    });
```

Comment:
What does this do? I don't quite get it

Reply:
Mock the behavior of reading the default private key to fail.

Reply:
Where is it used? And where do you assert the error message?

Reply:
Per discussion, this is caused by the misleading error message. Updated.